src/tests/Parser/Brewmaster/Energy.test.js

```diff
@@ -0,0 +1,112 @@
+import Energy from 'Parser/BrewmasterMonk/Modules/Features/Energy';
+import { processEvents } from './Fixtures/processEvents';
+import { EnergyFight, buffs } from './Fixtures/EnergyFight';
+
+describe('Brewmaster.Energy', () => {
```

Comment:
I'd prefer if the tests were based around requirements rather than per function; I think this would catch way more issues. For instance if you test `it('stacks multiple Haste gains properly', ...)` with the public API you could catch that `applybuff` doesn't do this right now.